base/linalg/bidiag.jl

```diff
@@ -40,6 +40,15 @@ function getindex{T}(A::Bidiagonal{T}, i::Integer, j::Integer)
     i == j ? A.dv[i] : (A.isupper && (i == j-1)) || (!A.isupper && (i == j+1)) ? A.ev[min(i,j)] : zero(T)
 end
 
+## structured matrix methods ##
+function Base.replace_in_print_matrix(A::Bidiagonal,i::Integer,j::Integer,s::AbstractString)
+  if A.isupper
```

Comment:
Heh. At the risk of having my commit access taken away, I totally use nano too when I'm in a terminal.

Formatting looks good now, thanks, but it does still need a rebase to fix a merge conflict.